test_requirements.txt

```diff
@@ -8,4 +8,5 @@ git+https://github.com/QCoDeS/pyvisa-sim.git
 lxml
 codecov
 asv
+ruamel.yaml
```

Comment:
If I understand this correctly this is not only a test requirement but a regular requirement. That means that it should be declared in the setup.py file, unfortunately that would probably install the package under it's pip name on top of the Conda package 😞